R/hash-df.R

```diff
@@ -25,7 +25,7 @@ parse_markdown <- function(x) {
   #message("markdownToHTML version")
   #res <- enc2utf8(markdown::markdownToHTML(text = x, fragment.only = T, encoding = "UTF_8"))
   #message("renderMarkdown version")
-  res <- markdown::renderMarkdown(file, output = NULL,
+  res <- markdown::renderMarkdown(output = NULL,
```

Comment:
I investigated locally and yes, it seems like the recent version of rmarkdown breaks this code. Can you try replacing it with the following? 

```r
  res <- markdown::mark(
    file = NULL,
    output = NULL,
    text = x,
    format = "html"
  )
```

Reply:
Yes, the test passes locally now. I've pushed this change to my branch.

Reply:
Thanks Seb, looks like you might need to resolve some merge conflicts?

Reply:
Is this okay now?

Reply:
Yep great!